website/docs/welcome/about-us.md

```diff
@@ -11,15 +11,13 @@ Check out our blog post [here](/blog/brief-history) for more details on our hist
 ## Contributors
 
 Core Team:
-* Badrish Chandramouli, Vasileios Zois, Ted Hart, Yoganand Rajasekaran, Tal Zaccai, Lukas Maas, Darren Gehring, Irina Spiridonova.
+* Badrish Chandramouli, Vasileios Zois, Ted Hart, Yoganand Rajasekaran, Tal Zaccai, Lukas Maas, Darren Gehring.
 * Email: garnetteam AT microsoft.com
 
 Project Partners:
 * Alan Yang, Pradeep Yadav, Alex Dubinkov, Venugopal Latchupatulla.
 * Knut Magne Risvik, Sarah Williamson, Narayanan Subramanian, Saurabh Singh, Padmanabh Gupta.
 * Kevin Montrose, Matt Tremblay.
-* Open-source contributors to [Garnet](https://github.com/microsoft/garnet/contributors).
+* Paulus Pärssinen.
```

Comment:
Thanks for the shout out 😊😅

Reply:
Thanks for your stellar contributions, and look forward to more ahead 👍